### ISSUE TITLE
NumberFormatException causes app to crash

### ISSUE THREAD

**Issue by daparker** [daparker/checkvalve#3]
Crash reported when an invalid integer was entered into the Port field of the **Add New Server** Activity.  Stack trace:

```
java.lang.NumberFormatException: unable to parse '3960765329' as integer
    at java.lang.Integer.parse(Integer.java:388)
    at java.lang.Integer.parseInt(Integer.java:372)
    at java.lang.Integer.parseInt(Integer.java:332)
    at com.dparker.apps.checkvalve.AddServerActivity$1.onClick(AddServerActivity.java:71)
    at android.view.View.performClick(View.java:2485)
    at android.view.View$PerformClick.run(View.java:9080)
    at android.os.Handler.handleCallback(Handler.java:587)
    at android.os.Handler.dispatchMessage(Handler.java:92)
    at android.os.Looper.loop(Looper.java:130)
    at android.app.ActivityThread.main(ActivityThread.java:3687)
    at java.lang.reflect.Method.invokeNative(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:507)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:625)
    at dalvik.system.NativeStart.main(Native Method)
```

The code for parsing port numbers and timeouts should be wrapped in a try/catch in each class which accepts these inputs.

**Comment by daparker** [comment]
Fixed in 2.0-development branch, commit [fef158c](https://github.com/daparker/checkvalve/commit/fef158c355e7bea164754dcafa051be5373f99e3#diff-7872e716dd94652070614e763709a98a).

**Comment by daparker** [comment]
Fixed in version 2.0.5 (commit [34af762](https://github.com/daparker/checkvalve/commit/34af762336dfbcff2ba9fb219062227ffbd6298c)).
